vignettes/Antimalarial_Resistance.Rmd

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
It would be nice to have a vignette, but it could be done at a later date. We shouldn't add empty vignettes to the repo

Reply:
+1, lets get this in with the PR, otherise as it isn't the "fun" bit it'll get forgotten!
It's also helpful for the reviewer to see a worked example in a vignette

Reply:
I've written a draft vignette for the antimalarial resistance feature that covers how to use the set_antimalarial_resistance() function to parameterise runs with resistance and gives examples using the early treatment failure functionality currently built. This may need updating as additional resistance outcomes (e.g. a partner drug resistant outcome like late clinical failure) are added, but should give us something to go with!